docs/content/guide/component.ngdoc

```diff
@@ -233,9 +233,13 @@ it upwards to the heroList component, which updates the original data.
 </file>
 
 <file name="heroDetail.js">
-  function HeroDetailController($scope, $element, $attrs) {
+  function HeroDetailController($scope) {
```

Comment:
I made the absolute minimum number of changes to make the unit tests pass. In the unit tests the `$componentController` is provided with a `$scope` value, but not with `$element` or `$attrs`.
The `$scope` service is not being used in the controller though, so it might as well be removed.

If I'm not wrong, a PR was merged the other day removing the requirement to always provide `$scope` in `$componentController` locals, so it can be removed from there as well.
I can update the PR if you like.